docs/tutorials/Date Fruit Classification using MindsDB and MongoDB.mdx

```diff
@@ -0,0 +1,162 @@
+2. Navigate to `Add` button in the editor and click on the dropdown arrow as highlighted below. Choose the `Upload File` option.
+
+![add-data](images/date-fruit-classification/add-data-1.png)
+
+3. Import the `.csv` file (here it is `Date_fruit.csv`), name the datasource table that is used to store the file (here it is `date_fruit_data`), and click on the `Save and Continue` button.
+
+![file-upload](images/date-fruit-classification/add-data-2.png)
+
+Now you can run queries directly on the file as if it were a table. Let’s preview the data that we’ll use to train our predictor.
+```sql
+SHOW TABLES FROM files;
+SELECT * FROM files.date_fruit_data LIMIT 10;
+```
+![run-queries](images/date-fruit-classification/check-data.png)
+
+### Connecting the Data
+After successful installation, it's now time to connect **MongoDB Compass** with **MindsDB Cloud** and create *Mongo database*. Follow the steps 👇
+1. Create a new connection in **MongoDB Compass** by changing the host value *from* `localhost:27017` *to* `cloud.mindsdb.com` in the **URI** section as highlighted below.
+
+![mongo-connect-1](images/date-fruit-classification/mongo-connect-1.png)
+
+2. Click on `Advanced Connection Options`, then go to `Authentication` tab and click on `Username/Password` button. Now, input the *Username* and *Password* details that is associated to your **MindsDB Cloud account**. After that, click on `Save & Connect` button which will further ask you to save your new connection with a name. (here it is `mindsdb`)
+
+![mongo-connect-2](images/date-fruit-classification/mongo-connect-2.png)
+
+3. Next, you will be directed to a page that will show all databases present in your **MindsDB Cloud** at the left panel along with the `MONGOSH` shell at the bottom. Further you can navigate towards `Databases` tab in the top middle section for more info.
+
+![mongo-connect-3](images/date-fruit-classification/mongo-connect-3.png)
+
+4. In order to use *MongoDB* database, we need to give permission to MindsDB through **MongoDB Compass**. For that, navigate to `MONGOSH` shell and switch to the *mindsdb* project.
+```nosql
+> use mindsdb
+< 'switched to db mindsdb'
+```
+![use-mindsdb](images/date-fruit-classification/use-mindsb.png)
+
+5. As MindsDB enables adding databases to the Mongo instance using the `db.databases.insertOne()` method, let's create a new *Mongo database* by adding a new connection along with the database credentials. This will  insert this new database in **MindsDB Cloud**.
+On a successful connection, you can see the newly created database on the left panel as highlighted below.
+```mongodb
+db.databases.insertOne({
+    name: "mongodb-test", // name of the database
+    engine: "mongodb", // database engine to use
+    connection_args: {
+            "port": 27017, // connection port
+            "host": "mongodb://cloud.mindsdb.com:27017", // connection host
+            "database": "files"  // connecting database
```

Comment:
You are using MindsDB's files database here, but you should use a Mongo database (instead of connecting MindsDB's files database to itself).

Ideally, you should have your Mongo database (for example, locally), then upload the data file to your Mongo database. And at this step, you should connect your (local) Mongo database to MindsDB - instead of connecting MindsDB's files database.

So here is the summary of the steps:
1. Create a Mongo database and upload your `Date fruit` data file there.
2. Connect MindsDB to Mongo Compass - you did it correctly.
3. In Mongo Compass, use the insertOne method to connect your Mongo database to MindsDB, like this:
```
db.databases.insertOne({
    name: "mongodb-test", // name of the database
    engine: "mongodb", // database engine to use
    connection_args: {
            "port": 27017, // connection port
            "host": "mongodb://localhost:27017", // connection host
            "database": "db_name"  // connecting database
    }
});
```

Reply:
Oh. Okay!